pkg/build/pipelines/ruby/install.yaml

```diff
@@ -0,0 +1,39 @@
+name: Install a ruby gem
+
+needs:
+  packages:
+    - busybox
+    - ca-certificates-bundle
+    - ruby-3.2
+
+inputs:
+  gem:
+    description: |
+      Gem name
+    required: true
+
+  version:
+    description: |
+      Gem version to install. This can be a version tag (v1.0.0)
+    required: true
+
+pipeline:
+  - runs: |
+      TARGET_DIR_BIN="${{targets.destdir}}/usr/bin"
+      TARGET_DIR_INSTALL="${{targets.destdir}}$(ruby -e 'puts Gem.default_dir')/"
+
+      mkdir -p "${TARGET_DIR_BIN}"
+      mkdir -p "${TARGET_DIR_INSTALL}"
+      
+      gem install \
+      --install-dir ${TARGET_DIR_INSTALL}  \
+      --bindir ${TARGET_DIR_BIN} \
+      --no-document \
+      --ignore-dependencies \
+      --local \
+      --verbose \
+      -v ${{inputs.version}} \
+      ${{inputs.gem}}
+      
+      cd ${{targets.destdir}}
+      rm -rf build_info cache extensions plugins
```

Comment:
This was my mistake, it should be
```sh
cd ${TARGET_DIR_INSTALL} && rm -rf build_info cache extensions plugins
```

Reply:
Or perhaps we turn this into a `ruby/clean` pipeline

Reply:
added a clean pipeline